src/common/groupConfigEntityAttributes.ts

```diff
@@ -0,0 +1,25 @@
+import { GroupConfiguration } from 'pnc-api-types-ts';
+
+import { TEntityAttributes } from 'common/entityAttributes';
+
+interface IExtendedGroupConfig extends GroupConfiguration {
+  buildConfigsCount: number;
+}
+
+export const groupConfigEntityAttributes = {
+  id: {
+    id: 'id',
+    title: 'ID',
+  },
+  name: {
+    id: 'name',
+    title: 'Name',
+    filter: {
+      operator: '=like=',
+    },
+  },
+  buildConfigsCount: {
```

Comment:
It's defined, but no used. Let's use it.

Reply:
Sorry that I was intended to ask you whether we need this column as well. Will try to implement it.

Reply:
I can see it's still missing?

Reply:
Sorry I forgot to push the commit([747fbda](https://github.com/project-ncl/pnc-web-ui-react/pull/251/commits/747fbdaa68bdc7d224606627f5c8128259d04efd)), pushed accordingly.

Reply:
Just found that the commit message was trimmed by mistake, will correct it.